src/drawer/index.unit.tsx

```diff
@@ -8,47 +8,48 @@ const defaultProps = {
 describe('Drawer', () => {
   it('Renders with a custom className', () => {
     const customClassName = 'custom-drawer'
-    const wrapper = shallow(<Drawer {...defaultProps} className={customClassName}>body</Drawer>)
+    const wrapper = shallow(
+      <Drawer {...defaultProps} className={customClassName}>
+        body
+      </Drawer>,
+    )
     expect(wrapper.hasClass(customClassName)).toBe(true)
   })
 
   it('Should call `onOpen` and `scrollLock` after open', () => {
     const onOpen = jest.fn()
-    const onChange = jest.fn()
+    const onTransitionEnd = jest.fn()
     const wrapper = mount(
-      <Drawer
-        {...defaultProps}
-        onChange={onChange}
-        onOpen={onOpen}
-      >
+      <Drawer {...defaultProps} onTransitionEnd={onTransitionEnd} onOpen={onOpen}>
         body
       </Drawer>,
     )
     const scrollLock = jest.spyOn(wrapper.instance(), 'scrollLock')
-    wrapper.instance().open()
-    wrapper.find('.scrollableContent').simulate('transitionEnd')
+    expect(onOpen).toHaveBeenCalledTimes(0)
+    expect(scrollLock).toHaveBeenCalledTimes(0)
+    wrapper.setProps({ open: true })
+    wrapper.find('.kirk-drawer-scrollableContent').simulate('transitionEnd')
```

Comment:
Avoid using simulate and call the instance if necessary

Reply:
Done